lib/ansible/modules/remote_management/oneview/oneview_ethernet_network.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_ethernet_network
+short_description: Manage OneView Ethernet Network resources
+description:
+    - Provides an interface to manage Ethernet Network resources. Can create, update, or delete.
+version_added: "2.4"
+requirements:
+    - hpOneView >= 3.1.0
+author:
+    - Felipe Bulsoni (@fgbulsoni)
+    - Thiago Miotto (@tmiotto)
```

Comment:
The more the merrier. Take that literally, because if you have 2 maintainers adding 'shipit' to a PR, the PR is automatically merged. That means you need at least three maintainers if one of the maintainer creates a PR.